Source/Scene/Cesium3DTileset.js

```diff
@@ -131,6 +134,31 @@ define([
         this._refineToVisible = defaultValue(options.refineToVisible, false);
 
         /**
+         * Whether the tileset should should refine based on a dynamic screen space error. Tiles that are further
+         * away will be rendered will lower detail that closer tiles. This improves performance by rendering fewer
+         * tiles and making less requests, but may result in a slight drop in visual quality for tiles in the distance.
+         *
+         * @see Fog
+         */
+        this.useDynamicScreenSpaceError = defaultValue(options.useDynamicScreenSpaceError, false);
+
+        /**
+         * A scalar that determines the density used to adjust the dynamic SSE.
+         *
+         * @see Fog.density
+         */
+        this.dynamicScreenSpaceErrorDensity = 0.00278;
```

Comment:
Here and below, what is the range and how does the scale work in general?  I don't think there is enough doc here for a user to have a intuitive sense of how to set this.

Can we also expose this parameters in the Sandcastle example...later they would be in the 3D Tiles inspector?